src/app/glider/glider-list/glider-list.page.ts

```diff
@@ -35,25 +43,27 @@ export class GliderListPage implements OnInit, OnDestroy, AfterViewInit {
     this.filtered = this.gliderService.filtered$.getValue();
     this.gliderService.filtered$.pipe(takeUntil(this.unsubscribe$)).subscribe((value: boolean) => {
       this.filtered = value;
-    })
+    });
 
     this.initialDataLoad();
   }
 
   private async initialDataLoad() {
-    let loading = await this.loadingCtrl.create({
+    const loading = await this.loadingCtrl.create({
       message: this.translate.instant('loading.loading')
     });
     await loading.present();
-    this.gliderService.getGliders({ limit: this.gliderService.defaultLimit, clearStore: true }).pipe(takeUntil(this.unsubscribe$)).subscribe(async (res: Glider[]) => {
-      // @hack for hide export item
-      setTimeout(async () => {
-        this.content.scrollToPoint(0, 48);
+    this.gliderService.getGliders({ limit: this.gliderService.defaultLimit, clearStore: true })
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe(async (res: Glider[]) => {
+        // @hack for hide export item
+        setTimeout(async () => {
+          this.content.scrollToPoint(0, 48);
```

Comment:
Maybe also add await like in flight-list?

Reply:
done